<!-- -->
<launch>
	<arg name="pathDensity" 					default="0.25" /> <!-- distance between each two waypoints-->
	<arg name="enableSmoothing" 				default="false" /> <!-- 1 or 0 -->
	<arg name="enableLaneChange" 				default="false" /> <!-- 1 or 0 -->
	<arg name="enableRvizInput" 				default="true" /> <!-- 1 or 0 -->
	<arg name="enableReplan"    				default="false" /> <!-- 1 or 0 -->
	<arg name="mapSource" 						default="2" /> <!-- Autoware=0, Vector Map Folder=1, kml file=2 -->
	<arg name="mapFileName" 					default="/home/user/maps/nic7f/inlab.kml" /> <!-- incase of kml map source -->	
	
<node pkg="way_planner" type="way_planner" name="way_planner" output="screen">
		
		<param name="pathDensity" 				value="$(arg pathDensity)" />
		<param name="enableSmoothing" 			value="$(arg enableSmoothing)" />
		<param name="enableLaneChange" 			value="$(arg enableLaneChange)" />
		<param name="enableRvizInput" 			value="$(arg enableRvizInput)" />
		<param name="enableReplan" 				value="$(arg enableReplan)" />
		<param name="mapSource" 				value="$(arg mapSource)" />
		<param name="mapFileName" 				value="$(arg mapFileName)" />
		
	    
	</node> 
	
</launch>